imagetest/test_suites/packagevalidation/package_test.go

```diff
@@ -4,22 +4,45 @@
 package packagevalidation
 
 import (
+	"bufio"
 	"bytes"
 	"os/exec"
+	"runtime"
 	"strings"
 	"testing"
 
 	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
 )
 
+// findUnique returns a slice of elements that are in arrayA but not in arrayB.
+func findUniq(arrayA, arrayB []string) []string {
+	unique := []string{}
+	for _, itemA := range arrayA {
+		if !contains(arrayB, itemA) {
```

Comment:
I think you can just do `strings.Contains` here instead of re-implementing `contains`, unless I'm missing something.

Reply:
Yeah that would work better

Reply:
I'm sorry, I didn't realize I mistyped. I meant `slices.Contains`, it does exactly what you're doing manually.

Reply:
Ah ok, rewrote to uses slices.Contains